test/util/cluster/cluster.go

```diff
@@ -293,14 +299,14 @@ func GetNodeSecureBootState(clients *testclient.ClientSet, nodeName string) (boo
 		return false, err
 	}
 
-	stdout, stderr, err := pod.ExecCommand(clients, runningPod, "cat", "/host/sys/kernel/security/lockdown")
+	stdout, _, err := pod.ExecCommand(clients, runningPod, "cat", "/host/sys/kernel/security/lockdown")
```

Comment:
why did you remove the stderr check. I still can failed for other reason

Reply:
This check is useless. Stderr will be always empty. 

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/cb3b24b87a4de9e50f2b684b3a602dd6e657a8a1/test/util/pod/pod.go#L113

Check [Goclient src](https://github.com/kubernetes/client-go/blob/6711973af1c55d544f08252297ebf6dab2031975/tools/remotecommand/remotecommand.go#L46)

If command completes with non zero exit code, then err will be not nil.